imjoy/runners/subprocess.py

```diff
@@ -175,20 +539,22 @@ def kill_plugin(engine, pid):
         del plugins[pid]
 
 
-async def kill_all_plugins(engine, ssid):
+@sio_on("reset_engine_plugins", namespace=NAME_SPACE)
+async def reset_engine(engine, sid, _):
```

Comment:
We register a handler that will be called from the main `reset_engine` handler.

Reply:
It seems other handler functions has the same name as the corresponding events, would it make more sense to use the same name here?

Reply:
Yeah, we can use the same name. I change names on the event a couple of times and forgot to change the function name.